build_container/build_container_windows.ps1

```diff
@@ -123,24 +123,24 @@ RunAndCheckError $env:TEMP\LLVM-win64.exe @("/S") $true
 AddToPath $env:ProgramFiles\LLVM\bin
 
 # NASM (preferred by some OSS projects over MS ml.exe, closer to gnu/intel syntax)
-$nasmVersion = "2.15.03"
+$nasmVersion = "2.15.05"
 DownloadAndCheck $env:TEMP\nasm-win64.zip `
                  https://www.nasm.us/pub/nasm/releasebuilds/$nasmVersion/win64/nasm-$nasmVersion-win64.zip `
-                 e598d1a9c98345f8436750f42d1f7c5d75ba739919eef37cd9ae8406e6a38802
+                 f5c93c146f52b4f1664fa3ce6579f961a910e869ab0dae431bd871bdd2584ef2
 Expand-Archive -Path $env:TEMP\nasm-win64.zip -DestinationPath C:\tools\
 AddToPath C:\tools\nasm-$nasmVersion
 
 # Python3 (do not install via msys2 or the MS store's flavors, this version follows Win32 semantics)
 DownloadAndCheck $env:TEMP\python3-installer.exe `
-                 https://www.python.org/ftp/python/3.8.5/python-3.8.5-amd64.exe `
-                 cd427c7b17337d7c13761ca20877d2d8be661bd30415ddc17072a31a65a91b64
+                 https://www.python.org/ftp/python/3.9.0/python-3.9.0-amd64.exe `
+                 fd2e2c6612d43bb6b213b72fc53f07d73d99059fa72c96e44bde12e7815073ae
 # python installer needs to be run as an installer with Start-Process
 RunAndCheckError "$env:TEMP\python3-installer.exe" @("/quiet", "InstallAllUsers=1", "Include_launcher=0", "InstallLauncherAllUsers=0") $true
-AddToPath $env:ProgramFiles\Python38
-AddToPath $env:ProgramFiles\Python38\Scripts
+AddToPath $env:ProgramFiles\Python39
```

Comment:
good catch!

Reply:
It would be nice to automate this at some point down the road.